app/frontend/components/_sign-table.scss

```
@@ -0,0 +1,57 @@
+.sign-table-row {
+  border-bottom: 1px #C2CDD1 solid;
+  padding: 8px 0;
+
+  a {
+    text-decoration: none;
+  }
+
+  &__header {
+    font-weight: bold;
+    font-size: 0.825rem;
+    color: #556970;
```

Rather than hardcoding colours here, it's preferable to use SASS variables. This helps us to have a sense of how broad our colour palette is, and allows for consistent reuse.


In this case, this shade of grey is quite new, but on `main` can be referred to with

```suggestion
    color: $neutral-gray;
```